### PR TITLE
Update storage_create_bucket_turbo_replication.rb

### PR DESCRIPTION
**Pull request by rebecca-pete**
@ddelgrosso1  Related to b/217259317.
In this sample, the recovery point objective or (RPO) is set, not turbo replication.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-cloud-ruby/issues) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea.
- [ ] Follow the instructions in [CONTRIBUTING](CONTRIBUTING.md). Most importantly, ensure the tests and linter pass by running `bundle exec rake ci` in the gem subdirectory.
- [ ] Update code documentation if necessary.

closes: #<issue_number_goes_here>

### CODE REVIEW

File: google-cloud-storage/samples/storage_create_bucket_turbo_replication.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def create_bucket_turbo_replication bucket_name:
                                   location: "ASIA",
                                   rpo:      "ASYNC_TURBO"
 
-  puts "Created bucket #{bucket.name} in #{bucket.location} with turbo replication set to #{bucket.rpo}."
+  puts "Created bucket #{bucket.name} in #{bucket.location} with the recovery point objective (RPO) set to #{bucket.rpo}."
```
Comment (SandeepTuniki) [inline google-cloud-storage/samples/storage_create_bucket_turbo_replication.rb:27]: Seems like Rubocop isn't happy with the line length. Can you split this into two lines?

Something like this should work:
```rb
puts "Created bucket #{bucket.name} in #{bucket.location} with "
         + "the recovery point objective (RPO) set to #{bucket.rpo}."
```

**Review by SandeepTuniki — CHANGES_REQUESTED** [review]

File: google-cloud-storage/samples/storage_create_bucket_turbo_replication.rb (hunk 1 above)
Comment (rebecca-pete) [reply google-cloud-storage/samples/storage_create_bucket_turbo_replication.rb]: Updating. Thanks, @SandeepTuniki !

**Review by rebecca-pete — COMMENTED** [review]
As of today, location parameter should be a value representing a predefined dual-region.

**Comment by bajajneha27** [comment]
Hi @rebecca-pete ,
The samples test case is failing and it seems to be related this change. I ran the samples tests on main branch, it works fine. Can you please take a look at it?

**Comment by SandeepTuniki** [comment]
@rebecca-pete Friendly ping here. Can you look at Neha's comment above please?

**Comment by bajajneha27** [comment]
Hi @rebecca-pete , we'll go ahead and close this PR for now. Please fell free to re-open it when necessary. Thank you.

**Review by ddelgrosso1 — APPROVED** [review]
LGTM

**Comment by rebecca-pete** [comment]
[SandeepTuniki](https://github.com/SandeepTuniki) would you please review the changes?
I ran through the pull request with Denis. It appeared there was an unrelated issue with files_test.rb. Thanks!

**Review by SandeepTuniki — APPROVED** [review]
sample(google-cloud-storage): Update storage_create_bucket_turbo_replication.rb

**Pull request merged**